src/commcare_cloud/commcare_cloud.py

```diff
@@ -121,7 +121,7 @@ def run_on_control_instead(args, argv, force_latest_code):
         )
     else:
         bash_commands_template = (
-            '{env}cd ~/commcare-cloud && source ~/.virtualenvs/${{CCHQ_VIRTUALENV:-cchq}}/bin/activate '
+            '{env}source ~/commcare-cloud/control/activate_venv.sh --fail-on-error'
```

Comment:
Currently `env` will always be non-empty (if `CCHQ_VIRTUALENV` is not set to an empty string) since `default_virtualenv` always has a value. I think some lines above need to change as well:

https://github.com/dimagi/commcare-cloud/blob/7a6622a601c7efc4e3810d73b5b67854384ef34f/src/commcare_cloud/commcare_cloud.py#L115-L116

Line 115 can be removed entirely since we do not support Python 2 anymore. Line 116 should change to
```py
    venv = os.environ.get("CCHQ_VIRTUALENV")
```

Reply:
Update: I forgot that the activation script appends `-3.10` to the value of `CCHQ_VIRTUALENV` unless that suffix is already there (or unless `BIONIC_USE_SYSTEM_PYTHON=true`), so my comment was wrong and therefore not a blocker. However, the cleanup would still be nice.

Reply:
Thanks. See 9e0ec6c10fffc0507a69a8eb4c5a5ce8d89a0296.

Offlined with Daniel and clarified that we still want to allow callers to pass a value in for `CCHQ_VIRTUALENV` if it is set, but agreed there is no need to provide a default since the `activate_venv.sh` script handles that, and the `ansible` venv has long been irrelevant.